lib/spark_api/models/newsfeed.rb

```diff
@@ -0,0 +1,15 @@
+module SparkApi
+  module Models
+    class Newsfeed < Base 
+      
+      extend Finders
+
+      self.element_name = 'newsfeeds'
+
+      def update!(arguments={})
+        connection.put( "/newsfeeds/#{self.Id}", arguments ).first
```

Comment:
Thanks, I noticed the `Savable` concern, but I was using it incorrectly. I meant to ask about it in the pr.